pkg/apis/controller/v1beta1/appwrapper.go

```diff
@@ -271,11 +271,15 @@ type AppWrapperStatus struct {
 	TotalMemory float64 `json:"totalmemory,omitempty"`
 
 	TotalGPU int64 `json:"totalgpu,omitempty"`
+
+	// Re-queueing state fields
+	RequeueingTimeInSeconds int `json:"requeueing-time-seconds,omitempty"`
```

Comment:
Could you add the documentation here so it'll be picked up with #452.

Reply:
Done!